src/Data/Uniques/amulet.lua

```diff
@@ -1056,32 +1059,37 @@ League: Breach
 Source: Drops in Esh Breach or from unique{Esh, Forked Thought}
 Upgrade: Upgrades to unique{Choir of the Storm} using currency{Blessing of Esh}
 Variant: Pre 3.16.0
+Variant: Pre 3.21.0
 Variant: Current
 Requires Level 40
 Implicits: 1
 {tags:jewellery_attribute}+(20-30) to Intelligence
-Trigger Level 12 Lightning Bolt when you deal a Critical Strike
+{variant:1,2}Trigger Level 12 Lightning Bolt when you deal a Critical Strike
 {tags:jewellery_attribute}+(10-15) to all Attributes
 {tags:mana}(10-20)% increased maximum Mana
 {variant:1}Critical Strike Chance is increased by Lightning Resistance
 {variant:2}Critical Strike Chance is increased by Overcapped Lightning Resistance
+{variant:3}{tags:jewellery_elemental}50% increased Lightning Damage
+{variant:3}Lightning Damage with Non-Critical Strikes is Lucky
```

Comment:
```suggestion
{tags:jewellery_attribute}+(10-15) to all Attributes
{variant:3}{tags:jewellery_elemental}50% increased Lightning Damage
{tags:mana}(10-20)% increased maximum Mana
{variant:1}Critical Strike Chance is increased by Lightning Resistance
{variant:2}Critical Strike Chance is increased by Overcapped Lightning Resistance
{variant:3}Lightning Damage with Non-Critical Strikes is Lucky
```
You missed this one.